accounting/payables/supplier_bills/ocr.rst

```diff
@@ -0,0 +1,39 @@
+=====================================================================
+Digitalize your Vendor Bills with Optical Character Recognition (OCR)
+=====================================================================
+
+Encoding your bills manually can be a time-consuming task. Being able to digitalize them and
+extract the data into your database automatically will allow you to reduce errors, save time and
+have an effortless process.
+
+Set up the feature
+==================
+
+Go to :menuselection:`Accounting --> Settings --> Bill Digitalization`, and choose whether the bills
+should be processed automatically or manually.
```

Comment:
We should change digitalization in the system if it's that :P that's very frenchie

Not your concern tho...